From a516b549693ea373c0e4cb0286337f2e0fcd7f3c Mon Sep 17 00:00:00 2001
From: Remi Collet <remi@remirepo.net>
Date: Thu, 14 Dec 2017 12:58:28 +0100
Subject: [PATCH] Fix #220 relax test to follow upstream change

---
 tests/150_Imagick_setregistry.phpt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/150_Imagick_setregistry.phpt b/tests/150_Imagick_setregistry.phpt
index eb2b055..1b96452 100644
--- a/tests/150_Imagick_setregistry.phpt
+++ b/tests/150_Imagick_setregistry.phpt
@@ -35,10 +35,10 @@ if (array_key_exists("temporary-path", $registry) == true) {
 
 try {
 	$tmpPath = Imagick::getRegistry("non-existent string");
-	echo "Expected exception not thrown.\n";
+	var_dump($tmpPath == false); // Since 6.9.9-26, no exception raised
 }
 catch (\ImagickException $ie) {
-	echo $ie->getMessage() . "\n";
+	var_dump(true);
 }
 
 
@@ -47,4 +47,4 @@ catch (\ImagickException $ie) {
 Temporary-path was empty at start.
 Temporary path was set correctly.
 Temporary path was listed correctly.
-Imagick::getRegistry exception (%s)
+bool(true)
